pkg/database/database.go

```diff
@@ -7,22 +7,18 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"log"
-	"time"
 
-	"github.com/XSAM/otelsql"
-	"github.com/avast/retry-go/v4"
 	"github.com/kubearchive/kubearchive/pkg/models"
 	_ "github.com/lib/pq"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-const (
-	resourceTableName        = "resource"
-	resourcesQuery           = "SELECT data FROM %s WHERE kind=$1 AND api_version=$2"
-	namespacedResourcesQuery = "SELECT data FROM %s WHERE kind=$1 AND api_version=$2 AND namespace=$3"
-	writeResource            = `INSERT INTO %s (uuid, api_version, kind, name, namespace, resource_version, cluster_deleted_ts, data) Values ($1, $2, $3, $4, $5, $6, $7, $8) ON CONFLICT(uuid) DO UPDATE SET name=$4, namespace=$5, resource_version=$6, cluster_deleted_ts=$7, data=$8`
-)
+type queryData struct {
```

Comment:
What about?

```suggestion
type DatabaseStatements struct {
```

Reply:
I like the name better but why having it exposed?

Reply:
I wrote it with capital letter out of habit. It does not need to be exposed.

On a second thought this is consumed outside this file, so it may well be "exposed" if the mysql and postgresql implementations were modules other than `package database`.

Reply:
what about `SQLStatements`? I updated to that, I think it's even more explicit.

As for the exposing the variable outside of the package, you are right, if we want to move the different driver db files outside we will need to expose them.

Reply:
Read nice to me. At the end of the day, they are called statements too.